doc/lsst.pipe.base/command-line-task-data-repo-howto.rst

```diff
@@ -0,0 +1,136 @@
+.. _command-line-task-data-repo-howto:
+
+#################################################################
+Using Butler data repositories and reruns with command-line tasks
+#################################################################
+
+:ref:`Command-line tasks <using-command-line-tasks>` use Butler data repositories for reading and writing data.
+Butler data repositories are specially-configured directories on a filesystem.
```

Comment:
This simple statement could use more emphasis: **a repository is a specially-configured directory**. This may or may not be the place to go into what "special configurations" are necessary for a directory to be anointed as a proper repository; if it's not, a (future) link would be very helpful.

Along similar lines, I wonder if this section could be renamed "Using command-line tasks to process data" and then, at the top, explain more about how/why LSST uses Butler repositories for this (and sometimes reruns).